konami/identifiers.md

```diff
@@ -0,0 +1,131 @@
+# Konami Identifiers
+
+This file describes the various identifiers and their formats that are used in products by Konami.
+
+## Sources
+
+* [bemaniwiki](http://bemaniwiki.com/)
+* Game/cabinet manuals
+* MAME
+* Various sources around the internet
+
+## Format
+
+The format of each identifier/ID is described as a perl/php/shell-style string with `$` denoting
+that a variable is resolved, e.g. `$a` "resolve variable a". Additionally, a more complex regex
+string is provided to constrain the format properly.
+
+### Variables and values
+
+The following variables are used across the different identifiers and can denote the listed values:
+
+* `$p`: Product
+  * `G`: Game
+* `$t`: Type of product
+  * `C`: Upgrade: software only
+  * `E`: Upgrade: hardware (conversion) kit + software
+  * `K`: Upgrade: offline/no eamuse software version
+  * `N`: ???
+  * `Q`: Dedicated cabinet assembly/main body
+  * `U`: "Utilities", e.g. DVD software update kit, replacement monitors
+  * `X`: Hardware/board ???
+* `$g`: [Gamecode](#gamecode)
+* `$d`: Destination, target locale/market
+  * `A`: Asia
+  * `C`: China
+  * `E`: Europe
+  * `J`: Japan
+  * `K`: Korea
+  * `U`: US
+  * `Y`: Indonesia
+* `$s`: Specification: Product type and game dependent, e.g. type of cabinet or hardware
+  * `A`, `B`, `C`, ... , `Z`
+* `$r`: Software revision. Modern games utilizing the extension part pinned this to the value `A`
+  * `A`, `B`, `C`, ...
+* `$e`: Extension/build date of the software
+  * Format: `yyyymmddnn`
+    * `yyyy` = year, e.g. `2020`
+    * `mm` = month, e.g. `11` for November
+    * `dd` = day of month, e.g. `03`
+    * `nn` = build number on the day, e.g. `00`, `01`, ...
+
+#### Gamecode
```

Comment:
This doesn't have to be a sub-header of `Variables and values`, I propose dedenting it one level.